HTTP Server.core/IConnection.cs

```diff
@@ -1,16 +1,14 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Net;
-using System.Net.Sockets;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace HTTP_Server.core
 {
-    public class Test
+    public interface IConnection
```

Comment:
Yes, it will be able to use files and sockets.  It could theoretically use other things, but those are the two that I plan on using.